src/app/payment/paymentMethod/PaymentMethodId.ts

```diff
@@ -33,6 +33,7 @@ enum PaymentMethodId {
     StripeGooglePay = 'googlepaystripe',
     StripeV3 = 'stripev3',
     Zip = 'zip',
+    Mollie = 'mollie',
```

Comment:
Can you please add this PaymentMethodId alphabetically?

Reply:
its done